frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/storage-tab/vm-wizard-storage-row.tsx

```diff
@@ -42,20 +47,7 @@ const menuActionRemove = (
 const getActions = (
   wizardNetworkData: VMWizardStorageWithWrappers,
   opts: VMWizardStorageRowActionOpts,
-) => {
-  const actions = [menuActionEdit];
-
-  if (
-    ![
-      VMWizardStorageType.PROVISION_SOURCE_DISK,
-      VMWizardStorageType.PROVISION_SOURCE_TEMPLATE_DISK,
-      VMWizardStorageType.V2V_VMWARE_IMPORT_TEMP,
-    ].includes(wizardNetworkData.type)
-  ) {
-    actions.push(menuActionRemove);
-  }
-  return actions.map((a) => a(wizardNetworkData, opts));
-};
+) => [menuActionEdit, menuActionRemove].map((a) => a(wizardNetworkData, opts));
```

Comment:
nit: Why not just create the resulting array directly? 

Imho, similar indirection is overused in recent kubevirt-plugin code-base. This particular case is simple, so I am fine with it here. But other similar occurrences will cause troubles when maintaining - new-comers will not be able to change/fix anything without deeper understanding of complex flows. Their changes will either introduce new bugs or mess in the general flow, deviating from original ideas when designing it. My strong preference is to follow KISS principle (Keep It Simple Stupid).

Reply:
I guess you are right, it made more sense before when there were more lines